src/services/document-service.js

```diff
@@ -342,7 +343,8 @@ class DocumentService {
       language: lastRevision.language,
       sections: lastRevision.sections,
       contributors,
-      tags: lastRevision.tags
+      tags: lastRevision.tags,
+      archived: lastRevision.archived
     };
   }
 }
```

Comment:
I'm kinda missing the `default-prop-types.js`. I guess we want to avoid console errors in the frontend.

Reply:
Course, overlooked this, thanks!
FYI I also corrected tags mapping for document revision restore, seems we were reintroducing null tags in the DB this way (and losing old ones)